Tests/Tests/SDImageCacheTests.m

```diff
@@ -559,18 +559,31 @@ - (void)test44DiskCacheMigrationFromOldVersion {
     NSFileManager *fileManager = [[NSFileManager alloc] init];
     config.fileManager = fileManager;
     
-    // Fake to store a.png into old path
+    // Fake to store a%@.png into old path
     NSString *newDefaultPath = [[[self userCacheDirectory] stringByAppendingPathComponent:@"com.hackemist.SDImageCache"] stringByAppendingPathComponent:@"default"];
     NSString *oldDefaultPath = [[[self userCacheDirectory] stringByAppendingPathComponent:@"default"] stringByAppendingPathComponent:@"com.hackemist.SDWebImageCache.default"];
     [fileManager createDirectoryAtPath:oldDefaultPath withIntermediateDirectories:YES attributes:nil error:nil];
-    [fileManager createFileAtPath:[oldDefaultPath stringByAppendingPathComponent:@"a.png"] contents:[NSData dataWithContentsOfFile:[self testPNGPath]] attributes:nil];
+    
+    // Create 100 files to Migrate
+    for (NSUInteger i = 0; i < 100; i++) {
+        NSString *fileName = [NSString stringWithFormat:@"a%@.png", @(i)];
+        [fileManager createFileAtPath:[oldDefaultPath stringByAppendingPathComponent:fileName] contents:[NSData dataWithContentsOfFile:[self testPNGPath]] attributes:nil];
+    }
+    
     // Call migration
     SDDiskCache *diskCache = [[SDDiskCache alloc] initWithCachePath:newDefaultPath config:config];
     [diskCache moveCacheDirectoryFromPath:oldDefaultPath toPath:newDefaultPath];
     
-    // Expect a.png into new path
-    BOOL exist = [fileManager fileExistsAtPath:[newDefaultPath stringByAppendingPathComponent:@"a.png"]];
-    expect(exist).beTruthy();
+    // Expect a%@.png into new path and oldDefaultPath is deleted
+    BOOL isDirectory = NO;
+    for (NSUInteger i = 0; i < 100; i++) {
+        NSString *fileName = [NSString stringWithFormat:@"a%@.png", @(i)];
+        BOOL newFileExist = [fileManager fileExistsAtPath:[newDefaultPath stringByAppendingPathComponent:fileName] isDirectory:&isDirectory];
+        expect(newFileExist).beTruthy();
+        expect(isDirectory).beFalsy();
+    }
+    BOOL oldDefaultPathExist = [fileManager fileExistsAtPath:oldDefaultPath];
```

Comment:
This is not needed (I don't gurantee that old directory is kept)....Anyway is OK